src/index.jsx

```diff
@@ -5,9 +5,7 @@ import DigitalTerrainWebApp from './components/DigitalTerrainWebApp';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <DigitalTerrainWebApp />
-  </React.StrictMode>,
+  <DigitalTerrainWebApp />,
```

Comment:
Any particular reason to remove strict mode? Just out of curiosity

Reply:
React does some silly things in strict mode. There was console warnings about it. Can read more about it here. https://reactjs.org/docs/strict-mode.html

Based on that documenation I don't think it'll apply here.